src/shapes/path_group.class.js

```diff
@@ -207,9 +206,14 @@
      * @return {Boolean} true if all paths are of the same color (`fill`)
      */
     isSameColor: function() {
```

Comment:
Fixed this function, but i'm more for deleting it or improve the patGroup._set method that as of now is useless.
(it checks if all the paths are same color and if yes it change all the paths filling, some sort of ancient delegated properties that as of now feels unfinished work).